console/src/components/Services/Data/TableBrowseRows/ViewTable.js

```diff
@@ -141,10 +141,10 @@ class ViewTable extends Component {
   };
 
   retrieveManualTriggers = tableName => {
-    const { dispatch } = this.props;
-    return !semverCheck('manualTriggers')
-      ? Promise.resolve()
-      : dispatch(fetchManualTriggers(tableName));
+    const { dispatch, serverVersion } = this.props;
+    return semverCheck('manualTriggers', serverVersion)
```

Comment:
Can you store `supportManualTriggers` in state?
So that if it is required again, we can directly use that value?